crates/hyperswitch_domain_models/src/router_request_types.rs

```diff
@@ -439,6 +441,8 @@ pub struct PaymentsSyncData {
 
     pub amount: MinorUnit,
     pub integrity_object: Option<SyncIntegrityObject>,
+    pub attempt_status: AttemptStatus,
+    pub intent_status: IntentStatus,
```

Comment:
why do we need these details at the connector level? we already have attempt status in router_data.status

Reply:
Removed!